cfme/tests/infrastructure/test_host_drift_analysis.py

```diff
@@ -109,7 +109,6 @@ def test_host_drift_analysis(appliance, request, a_host, soft_assert, set_host_c
         message="Waiting for Drift History count to increase",
         fail_func=appliance.server.browser.refresh
     )
-
     # check drift difference
     soft_assert(a_host.equal_drift_results(
         '{} (1)'.format(added_tag.category.display_name), 'My Company Tags', 0, 1),
```

Comment:
Question: Does it works for host?

Reply:
Ahh, no. I forgot to change it for host... I will add it in new PR.